corehq/apps/hqwebapp/static/hqwebapp/css/export.css

```diff
@@ -0,0 +1,3 @@
+tr.selected-for-sort {
+    background-color: #0000FF !important;
```

Comment:
Any suggestion on how to prevent this property from being overridden without having to use ```!important```?

Reply:
What causes it to be overridden? You can prevent it by making this selector more specific - like `#customize-export .table-striped tr.selected-for-sort` - but it's helpful to know what specific selector is causing it to be overriden (so you don't end up making a ridiculously specific, hard to read selector that wasn't necessary).

Reply:
Thought about this some more. The easiest way to deal with this is probably to not do your own style, but instead add a bootstrap class like `.info` to the row when it's selected - check out the [contextual classes](http://bootstrapdocs.com/v3.0.3/docs/css/#tables-contextual-classes) docs.

Reply:
yes, bootstrap has its own classes for styling / highlighting table rows that we should please use here.